### PR TITLE
FEAT: Add Explore, Watchlater, Playlist, History Screens

### PR DESCRIPTION
**Pull request by VishalPatil18**
### 📝 Description

Create Screens for pages like Explore, Watchlater, History, Playlists

### 🚀 New behavior

Add Pages - 
- Explore
- Watch Later
- History 
- Playlist

### 🖼 Preview Link

[https://vistream-react-version-6bnker3qt-vishalpatil18.vercel.app/watchlater](https://vistream-react-version-6bnker3qt-vishalpatil18.vercel.app/watchlater)

### 💣 Is this a breaking change?
- [ ] Yes
- [x] No

### 🔬 Is the code self-reviewed?
- [x] Yes
- [ ] No

### 📋 Is the code well-formatted?
- [x] Yes
- [ ] No

> NOTE: The site is not responsive yet, will work on responsiveness at the end.

### CODE REVIEW

File: src/screens/Home.jsx [hunk 1]
```diff
@@ -6,16 +6,26 @@ import {
...
   FeaturedVideos,
 } from "../components";
+import { Link } from "react-router-dom";
+import styles from "./Home.module.css";
 
 const Home = ({ cname }) => {
   return (
     <main className={cname}>
-      <FilterTags />
+      <FilterTags page={"homePage"} />
```
Comment (kart-c) [inline src/screens/Home.jsx:15]: Since this is a string you can avoid writing curly brace.

**Review by kart-c — APPROVED** [review]
Website design is amazing @VishalPatil18 
All the pages are functional
Really like the filter section in your explore page

**Review by AtulPant2704 — APPROVED** [review]
Great way of breaking code into multiple functions, variable naming and code formatting is also good made it really easy to review. Having an array for Tabs was really good, learnt something new from it.

File: src/screens/Home.jsx (hunk 1 above)
Comment (VishalPatil18) [reply src/screens/Home.jsx:15]: Sure, will correct it in the next PR.

**Pull request merged**